crates/msrv/lib/Cargo.toml

```diff
@@ -5,13 +5,12 @@ publish = false
 version = "0.0.0"
 
 [features]
-default = ["std"]
-std = [
-  "wasm-bindgen/std",
-  "js-sys/std",
-  "wasm-bindgen-futures/std",
-  "web-sys/std",
-  "wasm-bindgen-test/std",
+default = [
+  "wasm-bindgen/default",
+  "js-sys/default",
+  "wasm-bindgen-futures/default",
+  "web-sys/default",
+  "wasm-bindgen-test/default",
 ]
```

Comment:
Don't replace the `std` feature, that's still needed.

Reply:
Is it? I did a simple text search and couldn't find it being used anywhere.

Reply:
Its not used because it was under `default` before. If we intend to add more features (`msrv`) we have to test these separately, in which case it will be used.

Reply:
Or, if you want to, we can merge this as-is and re-add the `std` crate feature in #4278. Let me know how you want to proceed.

Reply:
Could you please do that? I have no idea how to set that up properly.

Reply:
Did it in https://github.com/rustwasm/wasm-bindgen/pull/4278/commits/e1d2358abab72afb030710ad90047a4f9d574239.

Reply:
Thank you!